### ISSUE TITLE
Validation of .deb should run fink-package-precedence

### ISSUE THREAD

**Issue by dmacks** [fink/fink#133]
f-p-p is traditionally run at the end of CompileScript, and causes build to abort before InstallScript runs, so no binary .deb archive is created if f-p-p detects a problem. But if f-p-p is enhanced to know about new binary-level problems, the only reliably way to detect them for %n-%v-%r that were released prior to that is to rebuild all of fink. But the .deb validator runs on loose .deb binary packages (not necessarily during the fink-driven build process), so having val (optionally) run f-p-p would make it easy to rescan the existing body of built binaries for newly-detectable problems.

**Comment by akhansen** [comment]
Would we want a flag (like --use-fpp) for the fink validator then?  That'd avoid changing the current behavior in existing packages.

**Comment by dmacks** [comment]
I hadn't thought about what the default behavior should be, but having a CLI boolean flag was the plan.

But thinking further, I wonder if instead (or at least as a short-term quick-fix) f-p-p should be taught to read .deb? That would keep fpp from being hoisted into a fink core function. Not that I oppose fink core helping maintain fpp itself, but might seem like a bigger step than just a new feature.

**Comment by akhansen** [comment]
Yeah, that makes sense.  f-p-p already assumes that fink is installed. :-)
